lib/routes/user/home/floating_actions_bar.dart

```diff
@@ -59,7 +59,7 @@ class FloatingActionsBar extends StatelessWidget {
                             )
                           : Icon(Icons.add),
                       Padding(padding: EdgeInsets.only(left: 8.0)),
-                      Expanded(child: AutoSizeText(hasBalance ? "CONNECT TO PAY" : "ADD FUNDS", style: theme.addFundsBtnStyle, maxLines: 1,))
+                      Expanded(child: AutoSizeText(hasBalance ? "CONNECT TO PAY" : "ADD FUNDS", style: theme.addFundsBtnStyle, maxLines: 1, overflow: TextOverflow.ellipsis),)
```

Comment:
Not sure ellipsis is a good solution here. Why can't the AutoSizeText scale to fit? The parent has a constraint on the width.